-------------------- Log Start | Mon Mar 25 02:30:48 2024 --------------------
platform: Linux-6.5.0-26-generic-x86_64-with-glibc2.35
data_path: /home/turbo/printer_data
is_default_data_path: False
config_file: /home/turbo/printer_data/config/moonraker.conf
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: 6f44945189dc46f7a55f40779c59d449
unix_socket_path: /home/turbo/printer_data/comms/moonraker.sock
software_version: v0.8.0-324-ga3e4dac
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/turbo/printer_data/logs/moonraker.log
python_version: 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
launch_args: /home/turbo/moonraker-env/bin/python /home/turbo/moonraker/moonraker/moonraker.py -d /home/turbo/printer_data
msgspec_enabled: False
uvloop_enabled: False

#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /home/turbo/printer_data/comms/klippy.sock

[authorization]
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	FE80::/10
	::1/128
cors_domains = 
	*.lan
	*.local
	*://localhost
	*://localhost:*
	*://my.mainsail.xyz
	*://app.fluidd.xyz

[octoprint_compat]

[history]

[update_manager]
channel = dev
refresh_interval = 168

[update_manager mainsail]
type = web
channel = stable
repo = mainsail-crew/mainsail
path = ~/mainsail

[update_manager moonraker-telegram-bot]
type = git_repo
path = ~/moonraker-telegram-bot
origin = https://github.com/nlef/moonraker-telegram-bot.git
env = ~/moonraker-telegram-bot-env/bin/python
requirements = scripts/requirements.txt
install_script = scripts/install.sh

[update_manager crowsnest]
type = git_repo
path = ~/crowsnest
origin = https://github.com/mainsail-crew/crowsnest.git
install_script = tools/install.sh

[update_manager moonraker-obico]
type = git_repo
path = ~/moonraker-obico
origin = https://github.com/TheSpaghettiDetective/moonraker-obico.git
env = /home/turbo/moonraker-obico-env/bin/python
requirements = requirements.txt
install_script = install.sh
managed_services = 
	moonraker-obico

[update_manager octoeverywhere]
type = git_repo
channel = beta
path = /home/turbo/octoeverywhere
origin = https://github.com/QuinnDamerell/OctoPrint-OctoEverywhere.git
env = /home/turbo/octoeverywhere-env/bin/python
requirements = requirements.txt
install_script = install.sh
managed_services = 
	octoeverywhere

[announcements]
subscriptions = 
	octoeverywhere

[update_manager timelapse]
type = git_repo
primary_branch = main
path = ~/moonraker-timelapse
origin = https://github.com/mainsail-crew/moonraker-timelapse.git
managed_services = klipper moonraker

[timelapse]
output_path = ~/timelapse
frame_path = ~/timelapse/movies

#################################################################
All Configuration Files:
/home/turbo/printer_data/config/moonraker.conf
/home/turbo/printer_data/config/moonraker-obico-update.cfg
/home/turbo/printer_data/config/octoeverywhere-system.cfg
#################################################################

Database Versions: pylmdb = 1.4.1, lmdb = (0, 9, 29)
Unsafe Shutdown Count: 21

System Info:

***python***
  version: (3, 10, 12, 'final', 0)
  version_string: 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]

***cpu_info***
  cpu_count: 2
  bits: 64bit
  processor: x86_64
  cpu_desc: Intel(R) Core(TM)2 Duo CPU     E8400  @ 3.00GHz
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 5891860
  memory_units: kB

***sd_info***

***distribution***
  name: Ubuntu 22.04.4 LTS
  id: ubuntu
  version: 22.04
  version_parts: {'major': '22', 'minor': '04', 'build_number': ''}
  like: debian
  codename: jammy
  release_info: {}
  kernel_version: 6.5.0-26-generic

***virtualization***
  virt_type: none
  virt_identifier: none

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
Module libcamera unavailble, import failed

Systemd unit moonraker.service:
unit_name: moonraker.service
is_default: True
manager: systemd
Properties:
**Requires=['network-online.target', 'system.slice', '-.mount', 'sysinit.target']
**After=['sysinit.target', 'network-online.target', '-.mount', 'system.slice', 'basic.target', 'systemd-journald.socket']
**SupplementaryGroups=['moonraker-admin']
**EnvironmentFiles=/home/turbo/printer_data/systemd/moonraker.env
**ExecStart=/home/turbo/moonraker-env/bin/python $MOONRAKER_ARGS
**WorkingDirectory=/home/turbo/moonraker
**FragmentPath=/etc/systemd/system/moonraker.service
**Description=API Server for Klipper SV1
**User=turbo

Systemd unit klipper.service:
unit_name: klipper.service
is_default: True
manager: systemd
Properties:
**Requires=['system.slice', 'sysinit.target', '-.mount']
**After=['basic.target', 'system.slice', 'systemd-journald.socket', 'network-online.target', '-.mount', 'sysinit.target']
**SupplementaryGroups=[]
**EnvironmentFiles=/home/turbo/printer_data/systemd/klipper.env
**ExecStart=/home/turbo/klippy-env/bin/python $KLIPPER_ARGS
**WorkingDirectory=/home/turbo/klipper
**FragmentPath=/etc/systemd/system/klipper.service
**Description=Klipper 3D Printer Firmware SV1
**User=turbo
Git Repo moonraker: Warnings detected:
  Repo has untracked source files: ['moonraker/components/timelapse.py']
Git Repo klipper: No warnings detected
Git Repo moonraker-telegram-bot: No warnings detected
Git Repo crowsnest: No warnings detected
Git Repo moonraker-obico: No warnings detected
Git Repo octoeverywhere: No warnings detected
Git Repo timelapse: No warnings detected
Klipper Version: v0.12.0-125-gbfb71bc2
2024-03-25 03:30:48,508 [machine.py:_find_public_ip()] - Detected Local IP: 192.168.1.211
2024-03-25 03:32:08,612 [machine.py:_find_public_ip()] - Detected Local IP: 192.168.1.211
2024-03-25 09:18:30,549 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.1.12
2024-03-25 09:18:30,549 [application.py:log_request()] - 101 GET /websocket (192.168.1.12) [_TRUSTED_USER_] 0.87ms
2024-03-25 09:18:30,549 [websockets.py:open()] - Websocket Opened: ID: 135512732994864, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36, Host Name: 192.168.1.211
2024-03-25 09:18:30,597 [application.py:log_request()] - 101 GET /websocket (192.168.1.12) [_TRUSTED_USER_] 0.65ms
2024-03-25 09:18:30,597 [websockets.py:open()] - Websocket Opened: ID: 135512732998128, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36, Host Name: ender.pl
2024-03-25 09:18:30,611 [websockets.py:_handle_identify()] - Websocket 135512732998128 Client Identified - Name: OctoEverywhere-BrowserAgent, Version: 1.0.0, Type: web
2024-03-25 09:18:30,640 [application.py:log_request()] - 101 GET /websocket (192.168.1.12) [_TRUSTED_USER_] 0.60ms
2024-03-25 09:18:30,640 [websockets.py:open()] - Websocket Opened: ID: 135512733005136, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36, Host Name: 192.168.1.211
2024-03-25 09:18:30,658 [websockets.py:_handle_identify()] - Websocket 135512732994864 Client Identified - Name: OctoEverywhere-BrowserAgent, Version: 1.0.0, Type: web
2024-03-25 09:18:30,724 [websockets.py:_handle_identify()] - Websocket 135512733005136 Client Identified - Name: mainsail, Version: 2.10.0, Type: web
2024-03-25 10:19:20,906 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.1.12
2024-03-25 16:44:15,744 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.1.12
2024-03-25 16:44:15,744 [application.py:log_request()] - 101 GET /websocket (192.168.1.12) [_TRUSTED_USER_] 0.95ms
2024-03-25 16:44:15,744 [websockets.py:open()] - Websocket Opened: ID: 135512759280080, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36, Host Name: ender.pl
2024-03-25 16:44:15,751 [websockets.py:_handle_identify()] - Websocket 135512759280080 Client Identified - Name: mainsail, Version: 2.10.0, Type: web
2024-03-25 16:44:17,102 [websockets.py:on_close()] - Websocket Closed: ID: 135512732998128 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 6.50
2024-03-25 16:44:17,103 [websockets.py:on_close()] - Websocket Closed: ID: 135512759280080 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 1.36
2024-03-25 16:46:12,985 [websockets.py:on_close()] - Websocket Closed: ID: 135512732994864 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 2.43
2024-03-25 16:46:12,985 [websockets.py:on_close()] - Websocket Closed: ID: 135512733005136 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 2.34
2024-03-25 17:44:20,994 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.1.12
2024-03-25 19:37:51,921 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.1.211
2024-03-25 19:37:51,922 [application.py:log_request()] - 101 GET /websocket (192.168.1.211) [_TRUSTED_USER_] 0.94ms
2024-03-25 19:37:51,922 [websockets.py:open()] - Websocket Opened: ID: 135512759483888, Proxied: True, User Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:123.0) Gecko/20100101 Firefox/123.0, Host Name: 192.168.1.211
2024-03-25 19:37:53,874 [websockets.py:_handle_identify()] - Websocket 135512759483888 Client Identified - Name: OctoEverywhere-BrowserAgent, Version: 1.0.0, Type: web
2024-03-25 19:37:54,280 [application.py:log_request()] - 101 GET /websocket (192.168.1.211) [_TRUSTED_USER_] 5.05ms
2024-03-25 19:37:54,281 [websockets.py:open()] - Websocket Opened: ID: 135512732876960, Proxied: True, User Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:123.0) Gecko/20100101 Firefox/123.0, Host Name: 192.168.1.211
2024-03-25 19:37:54,318 [websockets.py:_handle_identify()] - Websocket 135512732876960 Client Identified - Name: mainsail, Version: 2.10.0, Type: web
2024-03-25 19:38:54,749 [application.py:prepare()] - Upload Request Received from 192.168.1.211
Content-Type: multipart/form-data; boundary=---------------------------138655429627934645341556784935
2024-03-25 19:38:54,758 [application.py:post()] - Processing Uploaded File: printer.cfg
2024-03-25 19:38:54,763 [application.py:log_request()] - 201 POST /server/files/upload (192.168.1.211) [_TRUSTED_USER_] 13.65ms
2024-03-25 19:38:55,974 [klippy_connection.py:close()] - Error closing Klippy Unix Socket
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/klippy_connection.py", line 769, in close
    await self.writer.wait_closed()
  File "/usr/lib/python3.10/asyncio/streams.py", line 343, in wait_closed
    await self._protocol._get_close_waiter(self)
  File "/usr/lib/python3.10/asyncio/selector_events.py", line 862, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
ConnectionResetError: [Errno 104] Connection reset by peer
2024-03-25 19:38:55,974 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2024-03-25 19:38:55,975 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.info, Code: 503, Message: Klippy Disconnected
2024-03-25 19:38:55,982 [websockets.py:on_close()] - Websocket Closed: ID: 135512760633424 Close Code: 1000, Close Reason: None, Pong Time Elapsed: 6.58
2024-03-25 19:38:56,227 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2024-03-25 19:38:56,498 [server.py:add_log_rollover_item()] - 
Systemd unit klipper.service:
unit_name: klipper.service
is_default: True
manager: systemd
Properties:
**Requires=['-.mount', 'sysinit.target', 'system.slice']
**After=['network-online.target', 'sysinit.target', '-.mount', 'systemd-journald.socket', 'system.slice', 'basic.target']
**SupplementaryGroups=[]
**EnvironmentFiles=/home/turbo/printer_data/systemd/klipper.env
**ExecStart=/home/turbo/klippy-env/bin/python $KLIPPER_ARGS
**WorkingDirectory=/home/turbo/klipper
**FragmentPath=/etc/systemd/system/klipper.service
**Description=Klipper 3D Printer Firmware SV1
**User=turbo
2024-03-25 19:38:57,996 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 127.0.0.1
2024-03-25 19:38:57,997 [application.py:log_request()] - 101 GET /websocket (127.0.0.1) [_TRUSTED_USER_] 1.67ms
2024-03-25 19:38:57,998 [websockets.py:open()] - Websocket Opened: ID: 135512759354128, Proxied: False, User Agent: , Host Name: 0.0.0.0
2024-03-25 19:38:58,005 [websockets.py:_handle_identify()] - Websocket 135512759354128 Client Identified - Name: OctoEverywhere, Version: 2.10.8, Type: agent
2024-03-25 19:39:17,733 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2024-03-25 19:39:17,738 [websockets.py:on_close()] - Websocket Closed: ID: 135512759354128 Close Code: 1000, Close Reason: None, Pong Time Elapsed: 9.74
2024-03-25 19:39:18,993 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2024-03-25 19:39:19,743 [application.py:log_request()] - 101 GET /websocket (127.0.0.1) [_TRUSTED_USER_] 0.86ms
2024-03-25 19:39:19,743 [websockets.py:open()] - Websocket Opened: ID: 135512732997120, Proxied: False, User Agent: , Host Name: 0.0.0.0
2024-03-25 19:39:19,747 [websockets.py:_handle_identify()] - Websocket 135512732997120 Client Identified - Name: OctoEverywhere, Version: 2.10.8, Type: agent
2024-03-25 19:39:21,002 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2024-03-25 19:39:21,003 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2024-03-25 19:39:21,004 [klippy_connection.py:_check_ready()] - 
MCU Protocol error

This is frequently caused by running an older version of the
firmware on the MCU(s). Fix by recompiling and flashing the
firmware.

Your Klipper version is: v0.12.0-125-gbfb71bc2
MCU(s) which should be updated:
mcu: Current version v0.12.0-25-g99d7af87
PIS: Current version v0.11.0-219-g645a1b83
Up-to-date MCU(s):
<none>

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.

mcu 'PIS': Command format mismatch: query_adxl345 oid=%c rest_ticks=%u vs query_adxl345 oid=%c clock=%u rest_ticks=%u
2024-03-25 19:49:10,424 [machine.py:_find_public_ip()] - Failed to detect local IP address
2024-03-25 19:49:10,425 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://127.0.0.1
2024-03-25 19:49:20,451 [machine.py:_find_public_ip()] - Detected Local IP: 192.168.1.211
2024-03-25 19:49:20,460 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://192.168.1.211
